### PR TITLE
Life hacks

### PR DESCRIPTION
**Pull request by Natan515**
Some fixes to make life better

### CODE REVIEW

**Comment by Natan515** [comment]
Can u Marge pls? @Avanatiker

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt [hunk 1]
```diff
@@ -119,6 +119,10 @@ object DiscordRPC : Module() {
             LineInfo.DIMENSION -> {
                 InfoCalculator.dimension()
             }
+            LineInfo.TASK -> {
+                if (HighwayTools.isEnabled) "Making highways"
```
Comment (theredstoner) [inline src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt:123]: I think we can make this more useful, maybe something like:

"Building highways for X seconds (Y blocks/hour)"

or something like that. @Avanatiker thoughts?

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt [hunk 2]
```diff
@@ -119,6 +119,10 @@ object DiscordRPC : Module() {
             LineInfo.DIMENSION -> {
                 InfoCalculator.dimension()
             }
+            LineInfo.TASK -> {
```
Comment (theredstoner) [inline src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt:122]: TASK is vague maybe make it something more related to the highway module. Maybe HIGHWAY or something.

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt (hunk 1 above)
Comment (Natan515) [reply src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt:123]: Yes but i saw that it didn't show it in discord cus the line is kami blue is so short.

File: src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt (hunk 2 above)
Comment (Natan515) [reply src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt:122]: Ah, Highway Work

**Pull request merged**